package-configuration/linux-cli/test_cli_install.py

```diff
@@ -51,10 +51,14 @@ def test_full_cli_install(self):
         finally:
             shutil.rmtree(tempdir)
 
-    def test_install_from_source(self):
+    def test_install_from_source_with_requirements(self):
         tempdir = tempfile.mkdtemp()
+        temp_requirements_file = tempfile.NamedTemporaryFile(delete=True)
+        with open(temp_requirements_file.name, 'w') as requirements_file:
```

Comment:
well obviously the meaning of life is not in this code but I do prefer to
choose the cleaner solution
On Jul 20, 2015 21:54, "Nir Cohen" <notifications@github.com> wrote:

> In package-configuration/linux-cli/test_cli_install.py
> <https://github.com/cloudify-cosmo/cloudify-packager/pull/145#discussion_r35029948>
> :
>
> >          tempdir = tempfile.mkdtemp()
> > +        temp_requirements_file = tempfile.NamedTemporaryFile(delete=True)
> > +        with open(temp_requirements_file.name, 'w') as requirements_file:
>
> You didn't.. it just... doesn't really matter :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudify-cosmo/cloudify-packager/pull/145/files#r35029948>
> .
>